### ISSUE TITLE
Model is not predicting well after loading from localStorage

### ISSUE THREAD

**Issue by Mudasar-Makandar** [tensorflow/tfjs#4406]
I have build model with the following structure in **Node JS** using **TFJS@1.7.0**

```
 let model = tf.sequential({
      layers: [
        tf.layers.dense({inputShape: [34], units: 16, activation: 'relu'}),
        tf.layers.dense({units: 3, activation: 'softmax'}),
      ]
    });

    model.compile({
      optimizer: tf.train.adam(0.02),
      loss: 'categoricalCrossentropy',
      metrics: ['accuracy']
    });
```

Model is giving >90% accuracy. Model has been tested and it is predicting well according to it's accuracy.

I saved model in localStorage with **model.save("localstorage://model")**.

But when i load model with **model.loadLayersModel("localstorage://model")** in different page,
The performance of model has got down to 50%, it is predicting like **non-trained models**.

Any wrong in saving or loading model.

**Comment by Mudasar-Makandar** [comment]
Model saving and Loading only works in a single page. 
If moved from one page to another within the same domain, the performance of model decreases to 50% if load from localStorage.
